ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/CV_PATHFINDER_TESTS.yml

```diff
@@ -4,6 +4,8 @@ wan_mode: cv-pathfinder
 # Have all the router in the examples use eBGP as underlay routing protocol -
 # the default is "none" for WAN routers"
 underlay_routing_protocol: ebgp
+# the default is currently ibgp for WAN routers but need to change it soon.
+overlay_routing_protocol: none
```

Comment:
this is probably not needed now that we are setting it to None without the knob. Test this